### ISSUE TITLE
incompete dates on AO items default to XX00 dates

### ISSUE THREAD

**Issue by DuncanDHall** [internetarchive/openlibrary#6737]
Old issue raised by Jeff Kaplan in 2017 in Jira: https://webarchive.jira.com/browse/OL-317

When a book's archive.org/metadata lists publication date as `"[18--?]"` it shows up in openlibrary as specifically `"1800"`.

### Evidence / Screenshot (if possible)
https://archive.org/details/plaintalesfromth00kipliala
![image](https://user-images.githubusercontent.com/16824525/178803202-9d862aa3-29bd-45c0-9906-dcd7839bd65d.png)
https://archive.org/metadata/plaintalesfromth00kipliala
![image](https://user-images.githubusercontent.com/16824525/178803612-d3fbf995-bc3e-4897-afd8-8d3ee238afe5.png)
https://openlibrary.org/books/OL7197646M/Plain_tales_from_the_hills
![image](https://user-images.githubusercontent.com/16824525/178803232-c801b47b-2f06-456b-9ca5-aed3ae53c416.png)

### Proposal & Constraints
Interpret this form of date metadata differently, perhaps as "1800s"?

### Stakeholders
@JeffKaplan

**Comment by LeadSongDog** [comment]
Related discussions on edition date issues: #806 #2039 #2711 #3391 #3421 #3496 #3746 #5333 #6021 #6038

tldr: consensus is to convert to, store, and sort on EDTF formats
